### PR TITLE
Add missing metce namespace.

### PR DESCRIPTION
**Pull request by jkorosi**
I think this is a good candidate for next patch release IWXXM 2.1.2.

### CODE REVIEW

**Comment by braeckel** [comment]
Hi @jkorosi, can you summarize the changes?  As Choy mentioned there are many changes to many files (13000 lines changed over 72 files).  They all appear to be added lines rather than modifications so it is difficult to detangle the existing and new contents.  You might consider creating a new pull request with only your changes included since it appears the current PR includes both old and new.  Let me know if I can help with anything.

**Comment by blchoy** [comment]
Sorry Jan but I still cannot figure out the changes you would like to make.  So far 3.0.0RC2 works fine without resolution issue regarding the metce namespace.  In case you find your expected changes are still outstanding in this release candidate, just let me know.

**Comment by jkorosi** [comment]
You are right this is not needed for 3.0.0RC2, because rdf, skos and reg namespaces are already there and metce is not used in any rule. 

My proposed change is related only to 2.1.1 version only to enable validation of rules like

`
<sch:pattern id="SIGMET.SIGMET7">
    <sch:rule context="//iwxxm:SIGMET|//iwxxm:VolcanicAshSIGMET|//iwxxm:TropicalCycloneSIGMET">
        <sch:assert test="(if(empty(om:result/@nilReason) and exists(iwxxm:analysis)) then(not(exists(iwxxm:analysis//om:procedure/*[name() != 'metce:Process']))) else(true()))">SIGMET.SIGMET7: The procedure of a SIGMET analysis should be a metce:Process</sch:assert>
    </sch:rule>
</sch:pattern>`

The proposed change is

`diff \-\-git a/2.1.1/rule/iwxxm.sch b/2.1.1/rule/iwxxm.sch
index cbae2ba..7607442 100644
\-\-\- a/2.1.1/rule/iwxxm.sch
\+\+\+ b/2.1.1/rule/iwxxm.sch
\@\@ -9,6 +9,10 \@\@
    <sch:ns prefix="om" uri="http://www.opengis.net/om/2.0"/>
    <sch:ns prefix="gml" uri="http://www.opengis.net/gml/3.2"/>
    <sch:ns prefix="aixm" uri="http://www.aixm.aero/schema/5.1.1"/>
\+   <sch:ns prefix="metce" uri="http://def.wmo.int/metce/2013"/>
\+   <sch:ns prefix="rdf" uri="http://www.w3.org/1999/02/22-rdf-syntax-ns#"/>
\+   <sch:ns prefix="skos" uri="http://www.w3.org/2004/02/skos/core#"/>
\+   <sch:ns prefix="reg" uri="http://purl.org/linked-data/registry#"/>
    <sch:pattern id="METAR_SPECI.ARS1">
       <sch:rule context="//iwxxm:AerodromeRunwayState">
          <sch:assert test="(if(@allRunways eq 'true') then( empty(iwxxm:runway) ) else true())">METAR_SPECI.ARS1: When all runways are being reported upon, no specific Runway should be reported</sch:assert>
diff \-\-git a/2.1.1/rule/iwxxm-collect.sch b/2.1.1/rule/iwxxm-collect.sch
index 322d1d2..4e872a0 100644
\-\-\- a/2.1.1/rule/iwxxm-collect.sch
\+\+\+ b/2.1.1/rule/iwxxm-collect.sch
\@\@ -10,6 +10,10 \@\@
    <sch:ns prefix="om" uri="http://www.opengis.net/om/2.0"/>
    <sch:ns prefix="gml" uri="http://www.opengis.net/gml/3.2"/>
    <sch:ns prefix="aixm" uri="http://www.aixm.aero/schema/5.1.1"/>
\+   <sch:ns prefix="metce" uri="http://def.wmo.int/metce/2013"/>
\+   <sch:ns prefix="rdf" uri="http://www.w3.org/1999/02/22-rdf-syntax-ns#"/>
\+   <sch:ns prefix="skos" uri="http://www.w3.org/2004/02/skos/core#"/>
\+   <sch:ns prefix="reg" uri="http://purl.org/linked-data/registry#"/>
    <sch:pattern id="COLLECT.MB1">
       <sch:rule context="//collect:MeteorologicalBulletin">
          <sch:assert test="count(distinct-values(for $item in //collect:meteorologicalInformation/child::node() return(node-name($item))))eq 1">COLLECT.MB1: All meteorologicalInformation instances in MeteorologicalBulletin must be of the same type</sch:assert>

diff \-\-git a/2.1.1/rule/iwxxm-collect-codelists.sch b/2.1.1/rule/iwxxm-collect-codelists.sch
index c4d6ab7..62f20c2 100644
\-\-\- a/2.1.1/rule/iwxxm-collect-codelists.sch
\+\+\+ b/2.1.1/rule/iwxxm-collect-codelists.sch
\@\@ -10,6 +10,10 \@\@
    <sch:ns prefix="om" uri="http://www.opengis.net/om/2.0"/>
    <sch:ns prefix="gml" uri="http://www.opengis.net/gml/3.2"/>
    <sch:ns prefix="aixm" uri="http://www.aixm.aero/schema/5.1.1"/>
\+   <sch:ns prefix="metce" uri="http://def.wmo.int/metce/2013"/>
\+   <sch:ns prefix="rdf" uri="http://www.w3.org/1999/02/22-rdf-syntax-ns#"/>
\+   <sch:ns prefix="skos" uri="http://www.w3.org/2004/02/skos/core#"/>
\+   <sch:ns prefix="reg" uri="http://purl.org/linked-data/registry#"/>
    <sch:pattern id="COLLECT.MB1">
       <sch:rule context="//collect:MeteorologicalBulletin">
          <sch:assert test="count(distinct-values(for $item in //collect:meteorologicalInformation/child::node() return(node-name($item))))eq 1">COLLECT.MB1: All meteorologicalInformation instances in MeteorologicalBulletin must be of the same type</sch:assert>
`

But as far as I understand Enrico, old version of IWXXM will be replaced by newer one or at least in this case 3.0.0 will replace 2.1.1.
